shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -18,18 +16,6 @@
 
 namespace flutter {
 
-#if SHELL_ENABLE_METAL
-bool ShouldUseMetalRenderer() {
```

Comment:
Removing this is likely what broke the google3 test harness. Despite running on a version of iOS sim that is *required* to support metal - due to the lack of supported GPU passthrough (and in defiance of all that is holu), creation of the MTLDevice will fail.

Reply:
Makes sense, I'll fix it up. Thank you for looking at it!